source/DomainTheory/index.lagda

```diff
@@ -124,6 +124,14 @@ import DomainTheory.ScottModelOfPCF.PCF             -- (0)
 import DomainTheory.ScottModelOfPCF.PCFCombinators  -- (1)
 import DomainTheory.ScottModelOfPCF.ScottModelOfPCF -- (2)
 
+{- Topology (by Ayberk Tosun)
+
+0. The definition of the Scott topology of a dcpo
+
+import DomainTheory.Topology.ScottTopology          -- (0)
+
+-}
```

Comment:
```suggestion
-}

import DomainTheory.Topology.ScottTopology          -- (0)
```

Reply:
Seems like the `import` is commented out, so it didn't build?

Reply:
@HarrisonGrodin Ah I totally missed this. Thanks for pointing it out!